README.md

```diff
@@ -444,6 +444,7 @@ See the [API documentation](./docs/api.md) for all endpoints.
 
 ### Libraries
 
+- [Abso](https://github.com/lunary-ai/abso/blob/main/README.md#ollama) (OpenAI-compatible TypeScript SDK for any LLM provider)
```

Comment:
Possible to add this further down in the list? Maintainers will re-organize in a follow up change. Thanks so much for building this!

Reply:
Sure!